### PR TITLE
add ui-router-extras v0.0.14 with git auto-update, close #5569

### LINKED ISSUE #5569: [Request] Add ui-router-extras
**Library name:** ui-router-extras
**Git repository url:** https://github.com/christopherthielen/ui-router-extras.git
**npm package url(optional):** https://www.npmjs.com/package/ui-router-extras
**Official homepage:** http://christopherthielen.github.io/ui-router-extras/#/home
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/christopherthielen/ui-router-extras
**Watch 41
  Star 535
 Fork 120**
@LeaYeh   could u help me check this PR for #5569 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/ui-router-extras/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "lazy",
+    "future"
+  ],
+  "dependencies": {
+    "angular": "^1.2.0",
+    "angular-ui-router": "~0.2.8"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/https://github.com/christopherthielen/ui-router-extras.git"
```
Comment (LeaYeh) [inline ajax/libs/ui-router-extras/package.json]: the link is wrong.

**Comment by maruilian11** [comment]
@LeaYeh i have corrected the link, could you help me check it again? thank you~

**Comment by LeaYeh** [comment]
i think it's okay. @PeterDaveHello

**Comment by PeterDaveHello** [comment]
`release` is kind of dist folder, and seems all the files in it are welcome on cdn, let's use `**/*` in filemap.

**Comment by maruilian11** [comment]
@LeaYeh i have correct the `filemap`, could you help me check it again? thank you~

**Comment by LeaYeh** [comment]
yee? tests failed ?

**Comment by maruilian11** [comment]
@LeaYeh sorry, i should tag you after it test successfully...
i will do it next time! could you help me check it again? thank you!

**Comment by LeaYeh** [comment]
fine. i think it okay now. @PeterDaveHello

**Comment by christopherthielen** [comment]
huh, neat.

**Pull request merged**